examples/doc_classification.py

```diff
@@ -26,7 +24,7 @@
 ########## Settings
 ##########################
 set_all_seeds(seed=42)
-device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+device, n_gpu = initialize_device_settings(use_cuda=True, local_rank=-1, fp16=False)
```

Comment:
I see why you are adding this here for multi-gpu support, but I think we should keep the examples rather simple. Can you make local_rank=-1 and fp16=False default args of  initialize_device_settings? Then we don't need to list them in the examples.

Reply:
good idea, done.